napari/_vispy/vendored/volume.py

```diff
@@ -377,8 +377,35 @@
 
 ISO_FRAG_SHADER = FRAG_SHADER.format(**ISO_SNIPPETS)
 
+
+MINIP_SNIPPETS = dict(
```

Comment:
actually... I led you a little astray here.  If at all possible we should try not to touch `vendored/volume.py` (theoretical goal is to have it be deleted each time vispy incorporates something upstream).  In this case, you can add this new code to `napari/_vispy/volume.py` ... emulating what @sofroniewn did with `ATTENUATED_MIP_SNIPPETS`.

Reply:
makes sense, thanks!